appserver/PKGBUILD

```diff
@@ -52,52 +53,70 @@ package() {
 
   install -D conf/antisamy.xml \
     "${pkgdir}/opt/zextras/conf/antisamy.xml"
+
```

Comment:
Remove extra space

Reply:
Check now, I've put a space between install for readability as I was not able to distinguish lines